modules/apps/info/info-api/src/main/java/com/liferay/info/exception/InfoFormValidationException.java

```diff
@@ -57,6 +57,36 @@ public String getLocalizedMessage(Locale locale) {
 
 	}
 
+	public static class ExceedsMaxEntries extends InvalidInfoFieldValue {
+
+		public ExceedsMaxEntries(String objectDefinitionLabel) {
+			super(StringPool.BLANK);
+
+			_objectDefinitionLabel = objectDefinitionLabel;
+		}
+
+		@Override
+		public String getLocalizedMessage(Locale locale) {
+			return LanguageUtil.format(
+				locale,
+				"the-limit-of-guest-entries-for-object-definition-has-been-" +
```

Comment:
I think you don't need to have this message key hard-coded here, you can pass the `ObjectEntryCountException` in your class constructor and use `getMessageKey()` method.